clients/include/testing_trmm_strided_batched.hpp

```diff
@@ -136,6 +132,69 @@ hipblasStatus_t testing_trmm_strided_batched(const Arguments& argus)
         {
             unit_check_general<T>(M, N, batch_count, ldb, stride_B, hB, hB_copy);
         }
+        if(argus.norm_check)
+        {
+            hipblas_error = norm_check_general<T>(
+                'F', M, N, ldb, stride_B, hB_copy.data(), hB.data(), batch_count);
+        }
+    }
+
+    if(argus.timing)
+    {
+        hipStream_t stream;
+        status = hipblasGetStream(handle, &stream);
+        if(status != HIPBLAS_STATUS_SUCCESS)
+        {
+            hipblasDestroy(handle);
+            return status;
+        }
+
+        int runs = argus.cold_iters + argus.iters;
+        for(int iter = 0; iter < runs; iter++)
+        {
+            if(iter == argus.cold_iters)
+                gpu_time_used = get_time_us_sync(stream);
+
+            status = hipblasTrmmStridedBatchedFn(handle,
+                                                 side,
+                                                 uplo,
+                                                 transA,
+                                                 diag,
+                                                 M,
+                                                 N,
+                                                 &alpha,
+                                                 dA,
+                                                 lda,
+                                                 stride_A,
+                                                 dB,
+                                                 ldb,
+                                                 stride_B,
+                                                 batch_count);
+
+            if(status != HIPBLAS_STATUS_SUCCESS)
+            {
+                hipblasDestroy(handle);
+                return status;
+            }
+        }
+        gpu_time_used = get_time_us_sync(stream) - gpu_time_used;
+
+        ArgumentModel<e_side_option,
+                      e_uplo_option,
+                      e_transA_option,
+                      e_diag_option,
+                      e_M,
+                      e_N,
+                      e_lda,
+                      e_ldb,
+                      e_stride_b,
+                      e_batch_count>{}
+            .log_args<T>(std::cout,
+                         argus,
+                         gpu_time_used,
+                         trmm_gflop_count<T>(M, N, K),
+                         trmm_gbyte_count<T>(M, N, K),
+                         hipblas_error);
```

Comment:
see comment above, should be

    trmm_gflop_count<T>(M,N,side)
    trmm_gbyte_count<T>(M,N,side)